keys/src/multisig.rs

```diff
@@ -10,7 +10,7 @@ use nimiq_utils::key_rng::{CryptoRng, RngCore, SecureGenerate};
 use rand::Rng;
 use sha2::{self, Digest, Sha512};
 
-use crate::{KeyPair, PublicKey, Signature};
+use crate::{EdDSAPublicKey, KeyPair, Signature};
```

Comment:
Should we rename `KeyPair` and `Signature` to `EdDSA...`?

Reply:
Or `Ed25519...` like mentioned above?